### PR TITLE
[JENKINS-47368] 1.3: Regression: core-js config.js forces to have windows.location

### PR DESCRIPTION
**Pull request by scherler**
# Description

See [JENKINS-47368](https://issues.jenkins-ci.org/browse/JENKINS-47368).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-core-js/src/js/config.js [hunk 1]
```diff
@@ -10,9 +10,19 @@ const organization = blueocean.organization || {};
 // any all features added by ?features=SOMETHING,SOMETHING_ELSE
 const pfx = 'features=';
 const pfxlen = pfx.length;
-(window.location.href.split('?')[1] || '').split('&')
-        .forEach(p => p.startsWith(pfx)
-            && (p.substring(pfxlen).split(',').forEach(f => { features[f] = true; })));
+const condition = window
+    && window.location
```
Comment (kzantow) [inline blueocean-core-js/src/js/config.js]: this formatting for the whole code change is really strange

**Comment by michaelneale** [comment]
is this one needed for 1.3 directly? Generally better to go master and cherry pick from there, or have 2 PRs? 

cc @alvarolobato @vivek

**Comment by scherler** [comment]
@michaelneale see #1476

**Comment by scherler** [comment]
@kzantow I just stumbled upon https://github.com/prettier/prettier 

```
foo(
  reallyLongArg(),
  omgSoManyParameters(),
  IShouldRefactorThis(),
  isThereSeriouslyAnotherOne()
);
```

is the result of a long line wrapped, seems to me my prior formating is very close to it, so IMHO not weird formating at all, but I have already changed it.

**Comment by michaelneale** [comment]
@scherler tgis branch is being prepared for 1.3 release 

Only Vivek or I should merge into it. Is this necessary? I hope this doesn’t stuff up. Pl

**Comment by michaelneale** [comment]
Please done merge into release branches when a release is going on

**Pull request merged**